Source/Core/DolphinQt2/Debugger/MemoryViewWidget.cpp

```diff
@@ -152,31 +155,35 @@ void MemoryViewWidget::Update()
     switch (m_type)
     {
     case Type::U8:
-      update_values([](u32 address) {
-        const u8 value = PowerPC::HostRead_U8(address);
+      update_values([&is_mram](u32 address) {
```

Comment:
`is_mram` can just be taken by value, it's a simple primitive type. Ditto elsewhere.